### DISCUSSION TITLE
Time to Abandoned `Python-Jose` Recommendation in Favour of Supported Packages?

### DISCUSSION THREAD

**Discussion by PhysicallyActive** [fastapi/fastapi#11345] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from jose import JWTError, jwt
```

### Description

First of all: Thank you for this great package (I'm using it extensively nowadays and I'm in love with the official docs)! :)

I have a question related to the Security chapter in the official docs; particularly, [the chapter relating to JWT tokens](https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/). The docs recommend the use of `Python-Jose` for encoding/decoding JWT, specifically stating:

"This tutorial previously used [PyJWT](https://pyjwt.readthedocs.io/).

But it was updated to use Python-jose instead as it provides all the features from PyJWT plus some extras that you might need later when building integrations with other tools."

However, it appears from GH discussions in the `Python-Jose` repository that `Python-Jose` has been abandoned (last release close to 3 years ago and last commit ~1 year ago) - Issues [#332](https://github.com/mpdavis/python-jose/issues/332), [#340](https://github.com/mpdavis/python-jose/issues/340), and [#341](https://github.com/mpdavis/python-jose/issues/341).

Maybe it's time to get back to recommending `PyJWT` which is still supported and maintained?
Thoughts/Comments?

Thank you all :)

### Operating System

Linux

### FastAPI Version

0.110.0

### Pydantic Version

2.6.4

### Python Version

3.12

**Comment by PhysicallyActive** [comment]
Most likely a duplicate of #9587

**Comment by tiangolo** [comment, marked as answer]
Thanks! Indeed, this was just updated here by @estebanx64: https://github.com/tiangolo/fastapi/pull/11589 :rocket:
